Documentation/ci-nnstreamer/auto-unittest.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+##
+# @file auto-unittest.sh
+# @brief Auto-generate unit test result
+# @how to append this script to /etc/crontab
+#       $ sudo vi /etc/crontab
+#       30 * * * * www-data /var/www/html/nnstreamer/Documentation/ci-nnstreamer/auto-unittest.sh
+# @see      https://github.com/nnsuite/nnstreamer
+# @author   Sewon Oh <sewon.oh@samsung.com>
+# @param    None
+#
+
+# Set-up environements
+dirpath="$( cd "$( dirname "$0")" && pwd )"
+build_root="${dirpath}/../../"
+arch_type="x86_64 armv7l"
+echo -e "[DEBUG] dirpath : $dirpath"
+echo -e "[DEBUG] build_root : $build_root"
+
+##
+# @brief check if a command is installed
+# @param 1 the command name
+function checkDependency {
+    which "$1" 1>/dev/null || {
+      echo "Ooops. '$1' command is not installed. Please install '$1'."
+      exit 1
+    }
+}
+
+# Check dependency
+check_dependency gbs
+
+# Create result folder
+pushd ${build_root}
+if [[ -d Documentation/unittest_result ]]; then
+    rm -rf Documentation/unittest_result/*
+else
+    mkdir -p Documentation/unittest_result
+fi
+
+for arch in $arch_type
+do
+    # Gbs build for unit test
+    # Unit test for ${arch}
+    gbs build -A ${arch} --overwrite --clean > temp.txt
```

Comment:
Please verify the execution result of "gbs build" command enough.
Note that we aim to archive the two reports (1. unittest: common libraries, 2. unittest: test casese) via cron daemon.